lib/virt_autotest/utils.pm

```diff
@@ -257,37 +257,49 @@ sub reset_log_cursor {
 #welcome everybody to extend this function
 sub check_failures_in_journal {
```

Comment:
@Julie-CAO  Hi Julie, thanks for the enhancement. The logic of function check_failures_in_journal and check_failures_in_journal seems very complex now. Would you please add some comment explaining the overall design/idea of the two functions? It will help maintainers understand code easier in future.

Reply:
> The logic of function check_failures_in_journal and check_failures_in_journal seems very complex now. Would you please add some comment explaining the overall design/idea of the two functions?

Hi @alice-suse , which two functions? I can add some description at the beginning of `sub check_failures_in_journal()`

Reply:
oh, sorry, one function only. I noticed you added some explanation. It is much clearer now. Thanks.